provisioner/terraform/timings_internal_test.go

```diff
@@ -0,0 +1,112 @@
+package terraform
+
+import (
+	_ "embed"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/tools/txtar"
+
+	"github.com/coder/coder/v2/coderd/database"
+	"github.com/coder/coder/v2/provisionersdk/proto"
+)
+
+var (
+	//go:embed testdata/timings-aggregation/simple.txtar
+	inputSimple []byte
+	//go:embed testdata/timings-aggregation/init.txtar
+	inputInit []byte
+	//go:embed testdata/timings-aggregation/error.txtar
+	inputError []byte
+	//go:embed testdata/timings-aggregation/complete.txtar
+	inputComplete []byte
+	//go:embed testdata/timings-aggregation/incomplete.txtar
+	inputIncomplete []byte
+	//go:embed testdata/timings-aggregation/faster-than-light.txtar
+	inputFasterThanLight []byte
```

Comment:
How about using an `filepath.Glob` instead? Then new tests can be added by just dropping new files in the folder.

Reply:
I wanted to avoid too much magic. We also might want to add more members to the table test struct later which would then make it awkward.